crates/sui-faucet/src/faucet/simple_faucet.rs

```diff
@@ -87,6 +90,8 @@ impl SimpleFaucet {
         let mut pending = vec![];
 
         let (producer, consumer) = mpsc::channel(coins.len());
+        let (sender, receiver) = mpsc::channel::<(Uuid, SuiAddress, Vec<u64>)>(10000);
```

Comment:
We should configure the bound on the channel, as some multiple of the request batch size.